.drone.star

```diff
@@ -118,6 +118,7 @@ def testing(ctx):
         'image': 'webhippie/golang:1.13',
         'pull': 'always',
         'commands': [
+          './bin/ocis-hello server &',
```

Comment:
This is not needed for API testing, beside that if we would do that this should be a separate detached step.